webhook/contextual_defaulter.go

```diff
@@ -107,7 +107,10 @@ func (h *defaulterForType) Handle(_ context.Context, req admission.Request) admi
 	// Default the object
 	//
 	// orig: https://github.com/kubernetes-sigs/controller-runtime/blob/v0.13.1/pkg/webhook/admission/defaulter.go#L78-L83
-	obj.Default(req)
+	if err := obj.Default(req); err != nil {
```

Comment:
I'm curious - is it common in Go to test the "if err ..." code paths?

Reply:
It depends on if you care about the result or not. Typically any handled errors should be tested since it introduces a new branch.